src/vcpkg/paragraphs.cpp

```diff
@@ -14,9 +14,9 @@ static std::atomic<uint64_t> g_load_ports_stats(0);
 
 namespace vcpkg
 {
-    static Optional<std::pair<std::string, TextRowCol>> remove_field(Paragraph* fields, const std::string& fieldname)
+    static Optional<std::pair<std::string, TextRowCol>> remove_field(Paragraph* fields, StringView fieldname)
     {
-        auto it = fields->find(fieldname);
+        auto it = fields->find(fieldname.data());
```

Comment:
This is not OK because `data()` is not necessarily null terminated.

Reply:
I also did this in a previous PR. Using `StringView::to_string()` instead.